src/mobile.md

```diff
@@ -10,11 +10,30 @@ One of the challenges is finding apps that focus on making this efficient.
 Here are some options. See also:
 
 <!-- not a list, to avoid sphinx TOC bug -->
-<https://cone.tangential.info/wiki/Mobile-ledgers>\
 <https://plaintextaccounting.org/#ui-mobile>
 
+## Offline data entry apps
+These apps saves a local *ledger file, which can be synced via Syncthing. For Adroid app there are two candidates: Cone and NanoLedger. Here is the difference between them:
```

Comment:
saves -> save

Adroid app -> Android

Here is the difference between them: you might want to add "as of 2025-09" or similar, to help this page age better ?